Core/src/Surfaces/DiscSurface.cpp

```diff
@@ -209,21 +210,18 @@ Acts::Vector2 Acts::DiscSurface::localCartesianToPolar(
 }
 
 Acts::BoundToFreeMatrix Acts::DiscSurface::boundToFreeJacobian(
-    const GeometryContext& gctx, const FreeVector& parameters) const {
-  // The global position
-  const Vector3 position = parameters.segment<3>(eFreePos0);
-  // The direction
-  const Vector3 direction = parameters.segment<3>(eFreeDir0);
-
+    const GeometryContext& gctx, const Vector3& position,
+    const Vector3& direction) const {
   assert(isOnSurface(gctx, position, direction, BoundaryCheck(false)));
 
   // The measurement frame of the surface
   RotationMatrix3 rframeT =
       referenceFrame(gctx, position, direction).transpose();
+
   // calculate the transformation to local coordinates
-  const Vector3 posLoc = transform(gctx).inverse() * position;
-  const double lr = perp(posLoc);
-  const double lphi = phi(posLoc);
+  const Vector3 pos_loc = transform(gctx).inverse() * position;
+  const double lr = perp(pos_loc);
+  const double lphi = phi(pos_loc);
```

Comment:
```suggestion
  const Vector3 posLoc = transform(gctx).inverse() * position;
  const double lr = perp(posLoc);
  const double lphi = phi(posLoc);
```
is there a reason for nonCamelCase?